### PR TITLE
Bug fix - duplicate LogzioRegion helm set

### PR DESCRIPTION
**Pull request by ralongit**
- Fix for duplicated `logzio-k8s-telemetry.secrets.LogzioRegion` Helm set flag

### CODE REVIEW

File: resources-linux/k8s/datasource_logs_utils.bash [hunk 1]
```diff
@@ -84,7 +84,7 @@ function build_logzio_logs_region_helm_set {
     send_log_to_logzio "$LOG_LEVEL_DEBUG" "$message" "$LOG_STEP_TRACES" "$LOG_SCRIPT_TRACES" "$func_name" "$AGENT_ID" "$PLATFORM" "$SUB_TYPE" "$CURRENT_DATA_SOURCE"
     write_log "$LOG_LEVEL_DEBUG" "$message"
 
-    local helm_set=" --set logzio-logs-collector.secrets.LogzioRegion=$region"
+    local helm_set=" --set logzio-logs-collector.secrets.logzioRegion=$region"
```
Comment (8naama) [inline resources-linux/k8s/datasource_logs_utils.bash:87]: is it with lower letter `logzioRegion` on purpose? (I remember a ticket about something similar)

File: resources-mac/k8s/datasource_logs_utils.bash [hunk 2]
```diff
@@ -84,7 +84,7 @@ function build_logzio_logs_region_helm_set {
     send_log_to_logzio "$LOG_LEVEL_DEBUG" "$message" "$LOG_STEP_TRACES" "$LOG_SCRIPT_TRACES" "$func_name" "$AGENT_ID" "$PLATFORM" "$SUB_TYPE" "$CURRENT_DATA_SOURCE"
     write_log "$LOG_LEVEL_DEBUG" "$message"
 
-    local helm_set=" --set logzio-logs-collector.secrets.LogzioRegion=$region"
+    local helm_set=" --set logzio-logs-collector.secrets.logzioRegion=$region"
```
Comment (8naama) [inline resources-mac/k8s/datasource_logs_utils.bash:87]: is it with lower letter `logzioRegion` on purpose? (I remember a ticket about something similar)

**Review by 8naama — APPROVED** [review]
Looks good only a question

File: resources-linux/k8s/datasource_logs_utils.bash (hunk 1 above)
Comment (ralongit) [reply resources-linux/k8s/datasource_logs_utils.bash:87]: It was handled in the chart but not the agent.
Released in `logzio-logs-collector` v1.0.4

Comment (ralongit) [reply resources-linux/k8s/datasource_logs_utils.bash:87]: It was handled in the chart but not the agent.
Released in `logzio-logs-collector` v1.0.4

File: resources-mac/k8s/datasource_logs_utils.bash (hunk 2 above)
Comment (ralongit) [reply resources-mac/k8s/datasource_logs_utils.bash:87]: It was handled in the chart but not the agent.
Released in `logzio-logs-collector` v1.0.4

**Pull request merged**